CMake/CppcheckTargets.cmake

```diff
@@ -1,30 +1,21 @@
-# - Run cppcheck on c++ source files as a custom target and a test
+# * Run cppcheck on c++ source files as a custom target and a test
 #
-# include(CppcheckTargets)
-# add_cppcheck(<target-name> [UNUSED_FUNCTIONS] [STYLE] [POSSIBLE_ERROR] [FAIL_ON_WARNINGS]) -
-# Create a target to check a target's sources with cppcheck and the indicated options
-# add_cppcheck_sources(<target-name> [UNUSED_FUNCTIONS] [STYLE] [POSSIBLE_ERROR] [FAIL_ON_WARNINGS]) -
-# Create a target to check standalone sources with cppcheck and the indicated options
+# include(CppcheckTargets) add_cppcheck(<target-name> [UNUSED_FUNCTIONS] [STYLE] [POSSIBLE_ERROR] [FAIL_ON_WARNINGS]) -
+# Create a target to check a target's sources with cppcheck and the indicated options add_cppcheck_sources(<target-name>
+# [UNUSED_FUNCTIONS] [STYLE] [POSSIBLE_ERROR] [FAIL_ON_WARNINGS]) - Create a target to check standalone sources with
+# cppcheck and the indicated options
 #
-# Requires these CMake modules:
-# Findcppcheck
+# Requires these CMake modules: Findcppcheck
 #
 # Requires CMake 3.3 or newer (uses the 'function' command and IN_LIST option)
 #
-#
-# Original Author:
-# 2009-2010 Ryan Pavlik <rpavlik@iastate.edu> <abiryan@ryand.net>
-# http://academic.cleardefinition.com
+# Original Author: 2009-2010 Ryan Pavlik <rpavlik@iastate.edu> <abiryan@ryand.net> http://academic.cleardefinition.com
 # Iowa State University HCI Graduate Program/VRAC
 #
-# Addition:
-# 2011 Arnaud Gelas <arnaud_gelas@hms.harvard.edu>
-# Harvard Medical School
+# Addition: 2011 Arnaud Gelas <arnaud_gelas@hms.harvard.edu> Harvard Medical School
```

Comment:
Do we want this soft of reformatting of the comments? Can we disable it?

Reply:
We can try disabling comment re-formatting.